### PR TITLE
Movie service

### PR DESCRIPTION
**Pull request by xjurko**

### CODE REVIEW

**Review by phoenixday — APPROVED** [review]

File: api/src/main/java/cz/muni/fi/pa165/dto/MovieDto.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class MovieDto {
+    Long id;
+    String name;
+    String caption;
+//    Integer releaseYear;
```
Comment (phoenixday) [inline api/src/main/java/cz/muni/fi/pa165/dto/MovieDto.java:18]: why is it commented? :)

File: api/src/main/java/cz/muni/fi/pa165/facade/MovieFacade.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * @author juraj
+ */
+public interface MovieFacade {
+    Iterable<MovieDto> findOtherAlsoLikedMovies(Long movieId);
+
+    Optional<MovieDto> findMovieByName(String name);
+
+    Optional<MovieDto> findMovieById(Long id);
+}
```
Comment (phoenixday) [inline api/src/main/java/cz/muni/fi/pa165/facade/MovieFacade.java:17]: well, in eshop they also had in facade methods for create and delete ...

Comment (xjurko) [reply api/src/main/java/cz/muni/fi/pa165/facade/MovieFacade.java:17]: of course - this is not the final state of this interface - but it compiles and this PR contains some foundation work that others can build on - but yes there is mork work to be done on this in future PRs

File: api/src/main/java/cz/muni/fi/pa165/dto/MovieDto.java (hunk 1 above)
Comment (xjurko) [reply api/src/main/java/cz/muni/fi/pa165/dto/MovieDto.java:18]: no corresponding field in the MovieEntity yet

**Pull request merged**